Tests/MongoSwiftTests/MongoDatabaseTests.swift

```diff
@@ -7,6 +7,13 @@ final class MongoDatabaseTests: MongoSwiftTestCase {
         continueAfterFailure = false
     }
 
+    override func tearDown() {
```

Comment:
it seems that if a test fails a defer doesn't actually get executed.... I had a `defer db.drop` in the test below but whenever the test failed I would get an error that the collection still existed on the next run.